python/analytics/bottle.py

```diff
@@ -1225,26 +1240,29 @@ class TemplatePlugin(object):
 
     def apply(self, callback, context):
         conf = context['config'].get('template')
+
         if isinstance(conf, (tuple, list)) and len(conf) == 2:
             return view(conf[0], **conf[1])(callback)
-        elif isinstance(conf, str) and 'template_opts' in context['config']:
-            depr('The `template_opts` parameter is deprecated.') #0.9
+        if isinstance(conf, str) and 'template_opts' in context['config']:
+            depr('The `template_opts` parameter is deprecated.')  # 0.9
             return view(conf, **context['config']['template_opts'])(callback)
-        elif isinstance(conf, str):
+        if isinstance(conf, str):
             return view(conf)(callback)
-        else:
-            return callback
+
+        return callback
 
 
 #: Not a plugin, but part of the plugin API. TODO: Find a better place.
-class _ImportRedirect(object):
+class _ImportRedirect:
     def __init__(self, name, impmask):
         ''' Create a virtual package that redirects imports (see PEP 302). '''
         self.name = name
         self.impmask = impmask
+        # print(name)
         self.module = sys.modules.setdefault(name, imp.new_module(name))
+        # self.module = sys.modules.setdefault(name, types.ModuleType(name))
```

Comment:
I think you meant to remove these commented out lines

Reply:
yes